pkg/operator/controller/status.go

```diff
@@ -148,6 +151,10 @@ func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatus
 			"%d ingress controllers available, want %d",
 			ingressesAvailable, numIngresses)
 	}
+	if !versionsEqual {
+		progressingCondition.Message = fmt.Sprintf(
+			"current release version does not match configured release version: %s", version)
```

Comment:
Consider the context in which this message is read.  As soon as the condition has been updated, this becomes past tense ("did not match"), right? Alternatively, I believe `Upgraded to %s` would be sufficient.

Reply:
Good point. I'll update.

> Alternatively, I believe Upgraded to %s would be sufficient.

This code checks for a diff between configured and current release versions and updates the `Progressing` message to trigger a new `LastTransitionTime`. As I mentioned in https://github.com/openshift/cluster-ingress-operator/pull/200#issuecomment-482752004, I believe that PR should be updated to include a `Progressing` status condition message whenever a status version is updated.